apps/examples/testcase/le_tc/kernel/tc_libc_signal.c

```diff
@@ -413,11 +419,12 @@ static void tc_libc_signal_sigwait(void)
 	snd_ret = (int)waitpid(snd_pid, &snd_status, 0);
 	TC_ASSERT_NEQ_CLEANUP("waitpid", snd_ret, ERROR, task_delete(recv_pid));
 	TC_ASSERT_EQ_CLEANUP("sigwait_sender", snd_status, OK, task_delete(recv_pid));
-
+	sem_post(&sync_sem);
```

Comment:
I don't get why this is necessary. Please describe it in the commit description.
Same for others